documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1314,10 +1473,20 @@ Restart your Kafka Connect process to pick up the new JAR files.
 
 If you are working with immutable containers, see link:https://hub.docker.com/r/debezium/[{prodname}'s container images] for Zookeeper, Kafka, SQL Server and Kafka Connect with the SQL Server connector already installed and ready to run. You can also xref:operations/openshift.adoc[run {prodname} on Kubernetes and OpenShift].
 endif::community[]
+For details about deploying the {prodname} SQL Server connector, see the following topics:
+
+* xref:installing-debezium-mysql-connectors[]
+* xref:debezium-sql-server-connector-configuration-example[]
+* xref:adding-sql-server-connector-configuration-to-kafka-connect[]
+* xref:descriptions-of-debezium-sql-server-connector-configuration-properties[]
 
 ifdef::product[]
 To deploy a {prodname} SQL Server connector, install the {prodname} SQL Server connector archive, configure the connector, and start the connector by adding its configuration to Kafka Connect. 
 
+// Type: procedure
+// Title: Installing {prodname} SQL Server connectors
+[id="installing-debezium-mysql-connectors"]
```

Comment:
Not MySQL

Reply:
Oh! Good catch! I didn't notice that at all. Thank you!